packages/block-editor/src/components/inserter/index.js

```diff
@@ -120,7 +127,10 @@ class Inserter extends Component {
 		return (
 			<Dropdown
 				className="block-editor-inserter"
-				contentClassName="block-editor-inserter__popover"
+				contentClassName={ classnames(
+					'block-editor-inserter__popover',
+					{ 'is-from-toolbar': !isAppender }
```

Comment:
Open to suggestions for a better classname here than `is-from-toolbar`.

Reply:
`is-top-toolbar-inserter`, maybe?